src/test/java/Entities/IngredientTest.java

```diff
@@ -12,15 +16,46 @@
     @Test
     public void testName() {
         Ingredient ingredient = new Ingredient("my name", Collections.emptyList());
-
         assertEquals("my name", ingredient.name());
     }
 
     @Test
-    public void testTags() {
+    public void testTagsEmpty() {
         Ingredient ingredient = new Ingredient("", Collections.emptyList());
-
         assertEquals(0, ingredient.tags().size());
     }
 
+    @Test
+    public void testTagsValues() {
+        Tag tag1 = new Tag("Dairy");
+        Tag tag2 = new Tag("Gluten");
```

Comment:
This code is perfectly fine. Keep it as is if you want but here is the method I usually use:

With this, as more tests are added to a given function (`testTagsXYZ`, `testTagsABC`, etc.) it may be hard to find all of the cases we are testing since there are many functions that work on the `tags` function. 
So I prefer to combine all of these into  a single `testTags` function that covers all cases one after another. To me, this is more readable when looking at test results (though it loses some granularity, but this is often ok since the test will tell you which line within the function failed).

There are certainly arguments to be made both ways.

So again, this is just personal preference. Decide which you prefer.